setup.sh

```diff
@@ -1,20 +1,86 @@
-DATABASENAME=Glue
 
-which -s brew
-if [[ $? != 0 ]] ; then
-    # Install Homebrew
-    # https://github.com/mxcl/homebrew/wiki/installation
-    /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-else
-    brew update
-fi
+databaseName=Glue
 
-brew install mysql
-mysql.server restart
-mysql -uroot -e "CREATE DATABASE ${DATABASENAME} /*\!40100 DEFAULT CHARACTER SET utf8 */;"
 
-brew install maven
+lowercase(){
+	echo "$1" | sed "y/ABCDEFGHIJKLMNOPQRSTUVWXYZ/abcdefghijklmnopqrstuvwxyz/"
+}
 
-brew install node
+# Adapted from https://github.com/coto/server-easy-install/blob/master/lib/core.sh
+installPackages(){
+    OS=`lowercase \`uname\``
+	KERNEL=`uname -r`
+	MACH=`uname -m`
+    noSupport="Sorry your OS is not supported in this setup script, please follow the setup instructions in the README"
 
-brew install yarn
+    if [ "${OS}" == "windowsnt" ]; then
+        echo -n ${noSupport}
+	elif [ "${OS}" == "darwin" ]; then
+        which -s brew
+        if [[ $? != 0 ]] ; then
+            # Install Homebrew
+            # https://github.com/mxcl/homebrew/wiki/installation
+            /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+        else
+            brew update
+        fi
+
+        brew install mysql
+        mysql.server restart
+        mysql -uroot -p -e "CREATE DATABASE ${databaseName} /*\!40100 DEFAULT CHARACTER SET utf8 */;"
```

Comment:
one of the tests for this is my girlfriend (non-tech) has to be able to run this without asking me for help. i think we just need a bit more guidance here. maybe beforehand we print hit enter if you're not sure.